pyproject.toml

```diff
@@ -0,0 +1,369 @@
+[build-system]
+requires = ["setuptools~=68.0.0", "wheel~=0.40.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aiosupervisor"
+version = "0.1.0"
+license = { text = "Apache-2.0" }
+description = "Asynchronous python client for Home Assistant Supervisor."
+readme = "README.md"
+authors = [
+    { name = "The Home Assistant Authors", email = "hello@home-assistant.io" },
+]
+keywords = ["docker", "home-assistant", "api", "client-library"]
+requires-python = ">=3.12.0"
+dependencies = [
+    "aiohttp>=3.3.0,<4.0.0",
+    "mashumaro>=3.11,<4.0",
+    "orjson>=3.6.1,<4.0.0",
+    "yarl>=1.6.0,<2.0.0",
+]
+
+[project.urls]
+"Homepage" = "https://www.home-assistant.io/"
+"Source Code" = "https://github.com/home-assistant/python-supervisor-client"
+"Bug Reports" = "https://github.com/home-assistant/python-supervisor-client/issues"
+"Docs: Dev" = "https://developers.home-assistant.io/"
+"Discord" = "https://www.home-assistant.io/join-chat/"
+"Forum" = "https://community.home-assistant.io/"
+
+[tool.setuptools]
+platforms = ["any"]
+zip-safe = false
+include-package-data = true
+
+[tool.setuptools.packages.find]
+include = ["aiosupervisor*"]
+
+[tool.pylint.MAIN]
```

Comment:
I'd consider dropping pylint completely. Ruff does a lot already and we don't use any custom pylint plugins here.